src/base/datetime/calendar/body/test/accessibility.test.ts

```diff
@@ -0,0 +1,230 @@
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowUp" })
+      );
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.previousWeek);
+    });
+
+    it('should be highlight previous week when triggered "Down" keyboard event for IE', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(new KeyboardEvent("keydown", { key: "Down" }));
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.nextWeek);
+    });
+
+    it('should be highlight next week when triggered "ArrowDown" keyboard event', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowDown" })
+      );
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.nextWeek);
+    });
+
+    it('should be highlight next 2 weeks when triggered "ArrowDown" keyboard event', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowDown" })
+      );
+      await elementUpdated(el);
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowDown" })
+      );
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.next2Weeks);
+    });
+
+    it('should be highlight previous week when triggered "Left" keyboard event for IE', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
```

Comment:
@nmanhit 
I think don't need to `selectedEl.click()` before dispatch keydown event.
Please try remove it.